### ISSUE TITLE
Add newest Balancer v3 deployment

### ISSUE THREAD

**Issue by franzns** [balancer/backend#821]
Vault:

```
  "v3-vault-7": {
    "contracts": [
      {
        "name": "ProtocolFeeController",
        "address": "0x99f2D91EBA577e4Bf7175E72B3Ef2B6dDb1FaBe3"
      },
      {
        "name": "VaultAdmin",
        "address": "0x33156628B220fC3812b325Cd977CEB6523c01E2a"
      },
      {
        "name": "VaultExtension",
        "address": "0x2ed7a4eca0689C16A67E01dd25509fa231C148c2"
      },
      {
        "name": "Vault",
        "address": "0x7966FE92C59295EcE7FB5D9EfDB271967BFe2fbA"
      },
      {
        "name": "Router",
        "address": "0xDd10aDF05379D7C0Ee4bC9c72ecc5C01c40E25b8"
      },
      {
        "name": "BatchRouter",
        "address": "0x48341a762AD5470f96F4B553681696d1e5eeac0A"
      }
    ],
    "status": "ACTIVE"
  },
```

Pools factories:
```
  "v3-weighted-pool-7": {
    "contracts": [
      {
        "name": "WeightedPoolFactory",
        "address": "0x765ce16dbb3D7e89a9beBc834C5D6894e7fAA93c"
      },
      {
        "name": "MockWeightedPool",
        "address": "0x83a6375b50046256d47816Ff80B96975B0e589Cc"
      }
    ],
    "status": "ACTIVE"
  }
```
```
  "v3-stable-pool-7": {
    "contracts": [
      {
        "name": "StablePoolFactory",
        "address": "0x4b4b45Edf6Ca26ae894377Cf4FeD1FA9F82D85C6"
      },
      {
        "name": "MockStablePool",
        "address": "0x75457Be6462C35d23aBA13e43825079733acB27E"
      }
    ],
    "status": "ACTIVE"
  },
```

Hooks:
```
  "v3-hook-examples-7": {
    "contracts": [
      {
        "name": "FeeTakingHookExample",
        "address": "0xAD41a3ef49F2bdf7c02ab67e6a82b99121d0Bd98"
      },
      {
        "name": "ExitFeeHookExample",
        "address": "0x301EDe5Fd4f9d7266B09c3A2E38F97776447154B"
      },
      {
        "name": "DirectionalFeeHookExample",
        "address": "0xaf2692484AaE97c797686CaA468fb1B3E77F0bDC"
      },
      {
        "name": "LotteryHookExample",
        "address": "0x2371cB9AB7B498D7CD173cf30828ebc1430F1482"
      }
    ],
    "status": "ACTIVE"
  },
```

**Comment by franzns** [comment]
@mendesfabio can you provide the URL for the subgraph for the 7th deployment once its live?

**Comment by franzns** [comment]
subgraph urls can be found in the readme here https://github.com/balancer/balancer-subgraph-v3

**Comment by franzns** [comment]
@gmbronco if you can take care of this. You'll also need to remove any v3 related things from the canary DB, i.e. pools, swaps, hooks, add, removes, etc. I usually did it directly on the db.

**Comment by franzns** [comment]
waiting for version 8 

can be found here https://github.com/balancer/pvt-deployments/pulls once deployed

**Comment by franzns** [comment]
checkout v8 deployment addresses here https://github.com/balancer/pvt-deployments/pull/15/files#diff-10563202f0c611c9dbc1a814bd9c210731c0319b9cde48770c8b527f919eae75

**Comment by franzns** [comment]
@mendesfabio let us know once subgraph is ready

**Comment by mendesfabio** [comment]
I have shared with Jarek and updated the SG repository

- https://api.studio.thegraph.com/query/31386/balancer-v3-sepolia-8th/version/latest
- https://api.studio.thegraph.com/query/31386/balancer-pools-v3-sepolia-8th/version/latest

**Comment by gmbronco** [comment]
closed in #921
